File: Data Structures & Algorithms/remove-element/submission-0.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int j = nums.size()-1;
        int k = 0;
        for(int i = nums.size()-1; i >= 0; i--){
            if(nums[i] == val){
                nums[i] = nums[j];
                j--;
            } else {
                k++;
            }
        }
        return k;
    }
};

File: Data Structures & Algorithms/two-integer-sum/submission-0.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        myMap = {}
        for i in range(len(nums)):
            newTarget = target - nums[i]
            if newTarget in myMap:
                if(myMap[newTarget] != i):
                    return [min(i, myMap[newTarget]), max(i,myMap[newTarget])]
            myMap[nums[i]] = i

        return [0]

        